crypto/secp256k1/secp256k1_cgo.go

```diff
@@ -19,5 +20,33 @@ func (privKey PrivKeySecp256k1) Sign(msg []byte) ([]byte, error) {
 }
 
 func (pubKey PubKeySecp256k1) VerifyBytes(msg []byte, sig []byte) bool {
+	// if the signature is in the 65-byte [R || S || V] remove [V]
+	if len(sig) == 65 {
+		sig = sig[0:64]
+	}
 	return secp256k1.VerifySignature(pubKey[:], crypto.Sha256(msg), sig)
 }
+
+// SignRecoverAble creates an ECDSA signature on curve Secp256k1, a recoverable ECDSA signature .
+// it returns the result that is in 65-byte [R || S || V] (Ethereum signature format)
+func (privKey PrivKeySecp256k1) SignRecoverAble(msg []byte) ([]byte, error) {
+	rsv, err := secp256k1.Sign(crypto.Sha256(msg), privKey[:])
```

Comment:
this could be simplified to `returned secp256k1.Sign(crypto.Sha256(msg), privKey[:])`